diff --git a/block-db-go-bindings/go.mod b/block-db-go-bindings/go.mod
index fd58cbf..b296480 100644
--- a/block-db-go-bindings/go.mod
+++ b/block-db-go-bindings/go.mod
@@ -105,6 +105,6 @@ replace github.com/Fantom-foundation/lachesis-base => github.com/Fantom-foundati
 
 // This line is automatically uncommented by the `go-run-with-carmen-and-tracy.sh` script.
 //
-//replace github.com/0xsoniclabs/tracy => ../../tracy
+replace github.com/0xsoniclabs/tracy => ../../tracy
 
 //replace github.com/0xsoniclabs/tosca => ../../tosca
