diff --git a/block-db-go-bindings/cmd/block-db/app/state.go b/block-db-go-bindings/cmd/block-db/app/state.go
index d9b10d0..643e3d5 100644
--- a/block-db-go-bindings/cmd/block-db/app/state.go
+++ b/block-db-go-bindings/cmd/block-db/app/state.go
@@ -17,8 +17,9 @@ import (
 	"github.com/Fantom-foundation/lachesis-base/inter/idx"
 	"github.com/ethereum/go-ethereum/common"
 	"github.com/ethereum/go-ethereum/core/types"
+
 	// Uncomment to enable experimental Carmen features.
-	//_ "github.com/0xsoniclabs/carmen/go/experimental"
+	_ "github.com/0xsoniclabs/carmen/go/experimental"
 )
 
 // State is an abstraction of the Chain State Database. It tracks the balances,
diff --git a/block-db-go-bindings/go.mod b/block-db-go-bindings/go.mod
index d8cea2e..7f79e01 100644
--- a/block-db-go-bindings/go.mod
+++ b/block-db-go-bindings/go.mod
@@ -100,5 +100,6 @@ replace github.com/ethereum/go-ethereum => github.com/0xsoniclabs/go-ethereum v0
 replace github.com/Fantom-foundation/lachesis-base => github.com/Fantom-foundation/lachesis-base-sonic v0.0.0-20250701061954-44075d09185c
 
 // Enable local carmen and tracy modules if required for Tracy support.
-//replace github.com/0xsoniclabs/carmen/go => ../../carmen/go
-//replace github.com/0xsoniclabs/tracy => ../../tracy
+replace github.com/0xsoniclabs/carmen/go => ../../carmen/go
+
+replace github.com/0xsoniclabs/tracy => ../../tracy
