diff --git a/go/replay/state.go b/go/replay/state.go
index 693eabf..881c5c8 100644
--- a/go/replay/state.go
+++ b/go/replay/state.go
@@ -43,7 +43,8 @@ import (
 	"github.com/ethereum/go-ethereum/params"
 	"github.com/holiman/uint256"
 	// Uncomment to enable experimental Carmen features.
-	//_ "github.com/0xsoniclabs/carmen/go/experimental"
+
+	_ "github.com/0xsoniclabs/carmen/go/experimental"
 )

 // State is an abstraction of the Chain State Database. It tracks the balances,
diff --git a/go/go.mod b/go/go.mod
index 043657d..0baa98b 100644
--- a/go/go.mod
+++ b/go/go.mod
@@ -51,13 +51,13 @@ require (
 	github.com/consensys/gnark-crypto v0.20.1 // indirect
 	github.com/crate-crypto/go-eth-kzg v1.5.0 // indirect
 	github.com/davecgh/go-spew v1.1.1 // indirect
-	//github.com/crate-crypto/go-ipa v0.0.0-20240724233137-53bbb0ceb27a // indirect
+	github.com/crate-crypto/go-ipa v0.0.0-20240724233137-53bbb0ceb27a // indirect
 	github.com/deckarep/golang-set/v2 v2.9.0 // indirect
 	github.com/decred/dcrd/dcrec/secp256k1/v4 v4.4.1 // indirect
 	github.com/emicklei/dot v1.11.0 // indirect
 	github.com/ethereum/c-kzg-4844/v2 v2.1.7 // indirect
 	github.com/ethereum/go-bigmodexpfix v0.0.0-20250911101455-f9e208c548ab // indirect
-	//github.com/ethereum/go-verkle v0.2.3-0.20260102081149-aa0a270c0ed0 // indirect
+	github.com/ethereum/go-verkle v0.2.3-0.20260102081149-aa0a270c0ed0 // indirect
 	github.com/ferranbt/fastssz v1.0.0 // indirect
 	github.com/fsnotify/fsnotify v1.10.1 // indirect
 	github.com/getsentry/sentry-go v0.46.2 // indirect
@@ -121,7 +121,7 @@ replace github.com/Fantom-foundation/lachesis-base => github.com/Fantom-foundati

 // This line is automatically uncommented by the `go-run-with-carmen.sh` and `go-run-with-carmen-and-tracy.sh` scripts.
 //
-//replace github.com/0xsoniclabs/carmen/go => ../../carmen/go
+replace github.com/0xsoniclabs/carmen/go => ../../carmen/go

 // This line is automatically uncommented by the `go-run-with-carmen-and-tracy.sh` script.
 //
