diff --git a/go/go.mod b/go/go.mod
index 043657d..520a34d 100644
--- a/go/go.mod
+++ b/go/go.mod
@@ -125,6 +125,6 @@ replace github.com/Fantom-foundation/lachesis-base => github.com/Fantom-foundati

 // This line is automatically uncommented by the `go-run-with-carmen-and-tracy.sh` script.
 //
-//replace github.com/0xsoniclabs/tracy => ../../tracy
+replace github.com/0xsoniclabs/tracy => ../../tracy

 //replace github.com/0xsoniclabs/tosca => ../../tosca
